src/index.html

```diff
@@ -16,17 +16,23 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="body">
+    <div class="input input__line input__line--big">
+      <input
+        class="input__big"
+        type="text"
+        data-qa="keypress big"
+        placeholder="Try “Los Angeles“"
+      />
+    </div>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="input input__line input__line--small">
+      <input
+        type="text"
+        data-qa="hover small"
+        placeholder="Try “Los Angeles“"
+        class="input__small"
+      />
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. Consider renaming 'input__line--small' to something more meaningful.